traffic_portal/test/end_to_end/divisions/divisions-spec.js

```diff
@@ -67,11 +67,4 @@ describe('Traffic Portal Divisions Test Suite', function() {
 		expect(pageData.name.getText() === myNewDiv.name + ' updated');
 	});
 
-	it('should delete the new division', function() {
-		console.log('Deleting the new division: ' + myNewDiv.name + ' updated');
-		pageData.deleteButton.click();
-		pageData.confirmWithNameInput.sendKeys(myNewDiv.name + ' updated');
```

Comment:
Are you sure the problem isn't the `+ ' updated'` here?

Reply:
no, the delete works fine. the problem is that for these tests to work,  you can't delete certain things. 

i.e. a division is needed to create a region which is needed to create a phys location. therefore, divisions-spec and regions-spec can't have a 'delete' test.

Reply:
i will create another PR that does all the resource cleanup (deleting) at the end of all the tests.

Reply:
ah, okay